Distiller-server/src/main/java/com/mapr/distiller/server/processors/SystemCpuRecordProcessor.java

```diff
@@ -11,13 +11,13 @@ public boolean isAboveThreshold(SystemCpuRecord record, String metric,
 			String thresholdValue) throws Exception {
 
 		switch (metric) {
-		case "idle":
+		case "%idle":
 			if(record.getIdleCpuUtilPct() == -1d)
 				throw new Exception("Can not compare raw SystemCpuRecord to threshold");
 			else
 				return record.getIdleCpuUtilPct() > Double.parseDouble(thresholdValue);
 
-		case "iowait":
+		case "%iowait":
```

Comment:
I thought this would make a lot of cases to handle. Nevermind. I will merge this now.